packages/odyssey-storybook/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.9.2](https://github.com/okta/odyssey/compare/v0.9.1...v0.9.2) (2022-02-17)
+
+**Note:** Version bump only for package @okta/odyssey-storybook
```

Comment:
We updated this package to include updated stories. It's not a breaking or significant change, but it does go beyond a version bump.

Reply:
good catch 🤔 I'm wondering why our conventional commit toolchain [didn't include this commit?](https://github.com/okta/odyssey/pull/1342/commits/47c4fea1c293c132cab5b9263b838780627f7c85) I'll adjust manually for this release

Reply:
ah its because it was titled as `docs`. That was my suggestion in the future we can include those as `feat` scoped to `odyssey-storybook` for now lets not bother with updating the changelog and running thru CI again unless you feel strongly?